src/pages/mentees/[menteeId].tsx

```diff
@@ -67,7 +70,16 @@ function MenteeTabs({ mentee, mentorships }: {
         </TabPanel>
       )}
       <TabPanel>
-        <ChatRoom menteeId={mentee.id} />
+        <Flex direction={{ base: 'column', [breakpoint]: 'row' }}>
+          <Box order={{ base: 0, [breakpoint]: 2 }}>
+            <ChatRoom menteeId={mentee.id} />
+          </Box>
+          {sortedMentorships.map(m =>
+            <TabPanel key={m.id}>
+              <MentorshipPanel mentorship={m} />
```

Comment:
What I meant is to move `<ChatRoom menteeId={mentee.id} />` into `<MentorshipPanel>` and place it side-by-side with (on desktop) or on top of (on mobile) the <Transcripts> in <MentorshipPanel>.